.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    name: Run unit tests
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run tests inside docker container
+        uses: addnab/docker-run-action@v3
+        with:
+          image: manticoresearch/manticoresearch-buddy-test-kit:5.0.3
```

Comment:
Is it from ./Dockerfile?

Reply:
This is the prebuilt image that is in the company docker hub. It's built from Dockerfile, actually, almost once for future usage, I guess.